workshops/gloo-mesh-demo/README.md

```diff
@@ -54,12 +54,13 @@ Set these environment variables which will be used throughout the workshop.
 ```sh
 # Used to enable Gloo Mesh (please ask for a trail license key)
 export GLOO_MESH_LICENSE_KEY=<licence_key>
-export GLOO_MESH_VERSION=v2.1.0-beta29
+export GLOO_MESH_VERSION=v2.1.0-rc2
 
 # Istio version information
 export ISTIO_IMAGE_REPO=us-docker.pkg.dev/gloo-mesh/istio-workshops
 export ISTIO_IMAGE_TAG=1.13.8-solo
 export ISTIO_VERSION=1.13.8
+export ISTIO_REVISION=1-13
```

Comment:
"1-13" is hardcoded in the yamls

Reply:
I use it later when looking at the resources that were created via ILCM.

Reply:
my worry is that having this variable here might make people think they can just change the value here and have the rest of the workshop work

Reply:
Okay I'll take it out.